provisioning/tasks/build-composer.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Copy composer.json into place.
+  copy:
+    src: "{{ drupal_composer_json }}"
+    dest: "{{ drupal_composer_install_dir }}/composer.json"
+  when: drupal_composer_json is defined and not drupal_site.stat.exists
```

Comment:
This isn't needed for the usual setups (BLT or projects where Drupal VM is in vendor), but I added it as it could be useful in some scenarios. Eg. provisioning production environments.